searxng/searxng
# [simple theme] rework select and impove url

## What does this PR do?

This PR improves the simple theme:

* reworked select boxes in preferences
* moved search filter down underneath categories on result page
* added safe search as a filter; gets not saved to pref. (which happened on accident; but I actually think its the better option to not set the cookie every time someone just wants to change safe search once)
* make the article URL a flexbox consisting of scheme and domain and the second part is the path (the path gets cut of at the end of the line)
* replace / with > in the URL and drop params
* make URL 1rem and the h3 in article 1.2em
* fix the overlapping buttons when js is disabled in the search input
* padding around image previews changes to make videos, music and news category look better (maps looks a bit silly, but the whole category is kinda broken anyway)

## Why is this change important?

Polish up the simple theme and make the URL and h3 heading easier to read as well as cleanup the select in pref. and search header.

## How to test this PR locally?

```make run```

* make sure to test each cetegory with phone width and desktop width and also check pref. in dark and light theme

## Author's checklist

Desktop result page:
![image](https://user-images.githubusercontent.com/38733246/142654825-cfdb5f4c-6409-405e-acc0-3f921993ec63.png)

Mobile result page:
![image](https://user-images.githubusercontent.com/38733246/142654979-58e47d4a-7b9b-456f-8b4f-70fcddd4c80f.png)

Desktop pref. general:
![image](https://user-images.githubusercontent.com/38733246/142655084-3c8e997d-99b0-4864-9d24-40937c565596.png)

Desktop index with js disabled:
![image](https://user-images.githubusercontent.com/38733246/142655202-0311f351-c6fd-4d30-a41e-c93607475390.png)

Mobile result with js disabled:
![image](https://user-images.githubusercontent.com/38733246/142655374-fa368b22-1147-413f-b790-f3a23a57978b.png)


## Related issues

Closes #504, #372

Adding safesearch to search form #176
